packages/react-reconciler/src/ReactFiberDevToolsHook.old.js

```diff
@@ -84,15 +91,29 @@ export function onScheduleRoot(root: FiberRoot, children: ReactNodeList) {
   }
 }
 
-export function onCommitRoot(root: FiberRoot, priorityLevel: LanePriority) {
+export function onCommitRoot(root: FiberRoot, eventPriority: EventPriority) {
   if (injectedHook && typeof injectedHook.onCommitFiberRoot === 'function') {
     try {
       const didError = (root.current.flags & DidCapture) === DidCapture;
       if (enableProfilerTimer) {
-        const schedulerPriority =
-          priorityLevel === NoLanePriority
-            ? NormalPriority
-            : lanePriorityToSchedulerPriority(priorityLevel);
```

Comment:
Why inline? Do you anticipate this is the only place you'll need to do this conversion?

Reply:
Currently it is, yeah. Also there's a backlog item to discuss what kind of priority to expose and how. Scheduler priority doesn't really make sense anymore because of our decoupling work.

So inlining makes it harder for it to leak to more places.